### DISCUSSION TITLE
v0.5.0 | DeepL Support

### DISCUSSION THREAD

**Discussion by ThijmenDam** [ThijmenDam/BarTranslate#19] [Announcements]
Hi folks, this release includes a must requested feature!

- **Added support for DeepL Translate** 🚀
- Added a validator for the app's settings. If you accidentally delete the settings file, default settings are now provided as a fallback.
- Resolved a rare bug that prevented the settings panel from loading properly.

Download the latest release here: https://github.com/ThijmenDam/BarTranslate/releases/tag/v0.5.0

**Comment by korkmazonur** [comment]
Could not open BarTranslate on macOS ("BarTranslate is damaged and can't be opened") Macbook air M2
